src/pages/plugins/index.js

```diff
@@ -95,21 +101,28 @@ function PluginFooter({ keywords = [] }) {
 }
 
 function PluginComponent({ plugin }) {
+  const deprecatedCard = plugin.deprecatedMessage ? true : false;
+  const deprecatedMessage = deprecatedCard && plugin.deprecatedMessage;
+  const cardClasses = classnames('card', { [styles.pluginDeprecatedCard]: plugin.deprecatedMessage ? true : false });
+  const cardHeader = classnames( 'card__header', deprecatedCard ? styles.deprecatedCardHeader : styles.pluginCardHeader );
```

Comment:
`styles.pluginCardHeader` doesn't exist. Just use the pattern like you used for `cardClasses` instead of the ternary.

Reply:
Implemented with deleting pluginCardHeader.